src/components/Accordion/Accordion.tsx

```diff
@@ -7,14 +7,15 @@ type PropsAccordion = {
     description: string;
     isOpen?: boolean;
   }
+  key: number | string;
 }
 
-const Accordion = ({ data: { title, description, isOpen = false } }: PropsAccordion) => {
+const Accordion = ({ data: { title, description, isOpen = false }, key }: PropsAccordion) => {
   const [open, setOpen] = useState(isOpen);
   const descriptionRef = useRef(null);
 
   return (
-    <div className={styles.accordion}>
+    <div className={styles.accordion} key={key}>
```

Comment:
It is not necessary to receive the key as a prop, according to the react documentation, it is enough to put the key at component render time. I leave you the documentation, the first is the old documentation and the second is the latest.
[https://es.legacy.reactjs.org/docs/lists-and-keys.html](url)
[https://es.react.dev/learn/rendering-lists#recap](url)

Reply:
Thanks!